ui/js/components/auth.js

```diff
@@ -89,11 +90,11 @@ treeherder.component("login", {
             // Ask the back-end if a user is logged in on page load
             if (ctrl.userCanLogin) {
                 ThUserModel.get().then(async function (currentUser) {
-                    const userSession = localStorage.getItem('userSession');
-
-                    if (currentUser.email && userSession) {
+                    if (currentUser.email && localStorage.getItem('userSession')) {
+                        thNotify.record('User is authenticated in the back-end on page load', 'success');
                         ctrl.setLoggedIn(currentUser);
                     } else {
+                        thNotify.record('User is not authenticated in the back-end on page load', 'info');
```

Comment:
I think this one isn't quite right - it's really "user isn't authenticated in the backend or we had some race condition with localstorage". Perhaps the logging should differentiate between the reasons for logging in/out?